### PR TITLE
Sticky Table of Contents, Some Exercises/Evaluations, First Crack at Skip Lists

### LINKED ISSUE #4: Bloom Filter
There is not a page for a Bloom Filter. This is a good place to start. It uses the same two has functions as the Menagerie deck.

https://llimllib.github.io/bloomfilter-tutorial/

### LINKED ISSUE #6: Skip List
A Skip List can be created using the Acorn Cards.

https://en.wikipedia.org/wiki/Skip_list

### PR DESCRIPTION
**Pull request by qsebring3930**
I made the table of contents follow the page scroll, which actually was just a minima style setting. I added exercises and evaluations to a couple pages. Added the chits zip file you sent me to assets. Most important I took a crack at skip lists, my first attempt after finishing I realized that I misunderstood the algorithm so after some research I'm confident it's correct, just a bit wordy. Let me know what you think.

### CODE REVIEW

**Comment by qsebring3930** [comment]
I've also added the Algorithm/Example portions to the Skip list removal and Bloom Filter pages. If all looks well I'll record the videos for these pages.

**Comment by mgoadric** [comment]
I've been out on vacation last week, I'll get this reviewed this week.

**Comment by mgoadric** [comment]
The table of contents sticky looks great!

For Skip List, the algorithm looks good! I've been adding start and end columns to the list, that are as high as the highest column. They just make it easier for the search to start at the top of the start column and go right when necessary.

For the Bloom Filter, the algorithm looks sound too. I did not use the chits, but instead just flipped the cards to the white side to mark them. Either way is fine with me.

**Comment by qsebring3930** [comment]
I added head and tail columns to skip lists and recorded their videos, I also recorded the video for bloom filters, BST traversals, and BST rotations.
